README.md

```diff
@@ -676,6 +678,43 @@ convert nessus2hdf            Translate a Nessus XML results file into a Heimdal
     $ saf convert nessus2hdf -i nessus_results.xml -o output-hdf-name.json
 ```
 
+[top](#convert-other-formats-to-hdf)
+#### Microsoft Secure Score to HDF
+Output|Use|Command
+---|---|---
+Msft Combined json|Combine the outputs from secureScore and secureScoreControlProfile | `convert msft_secure2hdf -i <(jq -s \'{"secureScore": .[0], "profiles": .[1]}\' secureScore.json secureScoreControlProfiles.json) -o output-hdf-name.json`
```

Comment:
add the commands necessary to use the api to pull down the securescore and profiles via curl or something.

output|use|command
---|---|---
microsoft secure score json|this file contains blah blah security results info|`curl something or other`
microsoft secure score control profiles json|this file contains all this metadata|`curl something else`
combined json|combines the outputs together into one file for ease of processing|`jq stuff`